scripts/e2e-test.sh

```diff
@@ -192,6 +192,7 @@ done
 
 if [ "$generate_logs" -ne 0 ]; then
     if [ -n "$logsdir" ]; then
+        mkdir -p "$logsdir"
```

Comment:
not directly related but on the line below: `"$SCRIPTDIR"/e2e-cluster-dump.sh` - I find it very unusual to use quoting this way and looks unnatural to me. The whole string should be quoted either like this:
```
"${SCRIPTDIR}/something"
```
or like this
```
"$SCRIPTDIR/something"
```
does not matter in this case because the end of variable name is clearly delimited by the slash.

But if I'm the only one who finds this quoting pattern weird then feel free to ignore me (cc @arne-rusek ) :-)

Reply:
Personally I tend to quote whole string (as it looks nicer/more readable when syntax-highlighted to me), except when I don't e.g. when using glob: `"$SCRIPTDIR/"*` I am not aware of any repercussions doing it either way as long as variables are quoted and Having said that I would keep this consistent throughout the project, but just for the sake of being consistent (avoiding confusion in reader that is not bash-expert).

Reply:
fixed